torch_xla/csrc/tensor.cpp

```diff
@@ -1754,7 +1802,8 @@ XLATensor::CompilationResult XLATensor::Compile(
   return {/*device=*/coll.device,
           /*emitted_nodes=*/lowering_ctx.GetEmittedNodeCount(),
           /*computation=*/std::move(computations.front()),
-          /*parameters_data=*/std::move(po_data->parameters_data)};
+          /*parameters_data=*/std::move(po_data->parameters_data),
+          /*is_sharded=*/is_sharded};
```

Comment:
@alanwaketan I guess we need to add `is_sharded ` to `torch_xla_computation` in https://github.com/pytorch/xla/blob/master/torch_xla/csrc/computation.h#L119

Reply:
You mean add to upstream? I guess we can add that.